File: app/src/main/java/fr/insapp/insapp/models/EventInteraction.java
package fr.insapp.insapp.models;

import com.google.gson.annotations.SerializedName;

/**
 * Created by thomas on 19/07/2017.
 */

public class EventInteraction {

    @SerializedName("event")
    private Event event;

    @SerializedName("user")
    private User user;

    public EventInteraction(Event event, User user) {
        this.event = event;
        this.user = user;
    }

    public Event getEvent() {
        return event;
    }

    public User getUser() {
        return user;
    }
}

File: app/src/main/java/fr/insapp/insapp/models/EventInteraction.kt
package fr.insapp.insapp.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * Created by thomas on 19/07/2017.
 * Kotlin rewrite on 29/08/2019.
 */

@Parcelize
data class EventInteraction(
    @SerializedName("event") val event: Event,
    @SerializedName("user") val user: User
): Parcelable
